md2man/roff.go

```diff
@@ -109,9 +110,16 @@ func (r *roffRenderer) RenderNode(w io.Writer, node *blackfriday.Node, entering
 			out(w, strongCloseTag)
 		}
 	case blackfriday.Link:
-		if !entering {
```

Comment:
How come this was removed?

You'll have to forgive me I am really not familiar with a lot of this code anymore.

Reply:
Full access to the node is required to conditionally replace the link nodes text.
The walkAction is therefore set to blackfriday.SkipChildren

Reply:
Please check that it's working for you by running the tests.